android/include/mlops-agent-interface.h

```diff
@@ -0,0 +1,180 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * @file    mlops-agent-interface.h
+ * @date    5 April 2023
+ * @brief   A set of exported ml-agent interfaces for managing pipelines, models, and other service.
+ * @see     https://github.com/nnstreamer/deviceMLOps.MLAgent
+ * @author  Wook Song <wook16.song@samsung.com>
+ * @bug     No known bugs except for NYI items
```

Comment:
1. Provide a proper description.
2. If this provides the same interface as https://github.com/nnstreamer/deviceMLOps.MLAgent/blob/main/daemon/include/mlops-agent-interface.h , do NOT create another header. It will break future build when someone updates the interface. Share the SAME file.

Reply:
Okay, I will do that, thank you.